src/Root.js

```diff
@@ -18,20 +18,38 @@ console.disableYellowBox = true; // eslint-disable-line no-console
 // }
 // /* ------------------------ */
 
-const Root = () => (
-  <ErrorBoundary>
-    <ReduxProvider store={store}>
-      <VertxProvider>
-        <ThemeProvider>
-          <LayoutProvider>
-            <GoogleProvider>
-              <App />
-            </GoogleProvider>
-          </LayoutProvider>
-        </ThemeProvider>
-      </VertxProvider>
-    </ReduxProvider>
-  </ErrorBoundary>
-);
+class Root extends Component {
+  componentDidMount() {
+    this.attachStoreToWindow();
+  }
+
+  componentDidUpdate() {
+    this.attachStoreToWindow();
+  }
+
+  attachStoreToWindow() {
+    const storeFromRedux = store.getState();
+
+    return ( window.storeFromRedux = storeFromRedux );
```

Comment:
We'll run into an issue here in React Native where `window` is not defined. Are we able to do a `typeof window !== 'undefined'` check here before using the window object?

Reply:
Sounds good, but are we going to use `native` stuffs anymore ?